in-viewport.js

```diff
@@ -162,8 +168,8 @@ function createInViewport(container) {
       watching.push(elt);
     }
 
-    return function() {
-      watches.push(function() {
+    return function () {
```

Comment:
It tried uniformize the code style more than change it, but you're right you write anonymous fonction without space. I will Fix it ;)